apps/antalmanac/src/components/RightPane/SectionTable/EnrollmentHistoryPopup.tsx

```diff
@@ -0,0 +1,104 @@
+import { useState, useEffect, useMemo } from 'react';
+import { LineChart, Line, CartesianGrid, ResponsiveContainer, XAxis, YAxis, Tooltip, Legend } from 'recharts';
+import { Box, Link, Typography, Skeleton, useMediaQuery } from '@mui/material';
+import { MOBILE_BREAKPOINT } from '../../../globals';
+import { DepartmentEnrollmentHistory, EnrollmentHistory } from '$lib/enrollmentHistory';
+import { isDarkMode } from '$lib/helpers';
+
+export interface EnrollmentHistoryPopupProps {
+    department: string;
+    courseNumber: string;
+}
+
+const EnrollmentHistoryPopup = ({ department, courseNumber }: EnrollmentHistoryPopupProps) => {
```

Comment:
Purely stylistic preference; I like regular function syntax because it's similar to syntax in other languages and easier to read for me. I usually reserve arrow functions for callbacks that benefit from a short syntax or need to be unbound.

```suggestion
export function EnrollmentHistoryPopup({ department, courseNumber }: EnrollmentHistoryPopupProps) {
```

Also, another personal preference I have is using named exports over default exports because my LSP finds named exports better than default exports lol.